recipes/happy/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "happy" %}
+{% set version = "1.19.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - ghc
+    - zlib
```

Comment:
zlib and gmp can be removed here and cobal could probably be moved to the host section